### PR TITLE
RELATED: ONE-3178 Set formatter in xAxis for bar chart rotated by 90

### PR DESCRIPTION
**Pull request by ZdenekVidensky**

### CODE REVIEW

**Comment by ZdenekVidensky** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/386/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1628/ : SUCCESS in 2m 14s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/665/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/387/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/535/ : SUCCESS in 18m 16s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/388/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1629/ : SUCCESS in 2m 07s

File: src/components/visualizations/utils/common.ts [hunk 1]
```diff
@@ -49,6 +49,10 @@ export function getAttributeElementIdFromAttributeElementUri(attributeElementUri
     return attributeElementUri.slice(attributeElementUri.lastIndexOf(match) + match.length);
 }
 
+export function isRotationInRange(rotation: number, min: number = 75, max: number = 105) {
```
Comment (pbenes) [inline src/components/visualizations/utils/common.ts]: when function is called `isRotationInRange`, I would definitely not have defaults here and would always pass them.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/389/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1630/ : SUCCESS in 1m 47s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/72/ : SUCCESS in 19s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1635/ : SUCCESS in 1m 43s
- https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/666/ : SUCCESS in 15m 45s

**Pull request merged**